google-cloud-datastore/src/test/java/com/google/cloud/datastore/DatastoreOptionsTest.java

```diff
@@ -48,6 +64,16 @@ public void setUp() {
             .setServiceRpcFactory(datastoreRpcFactory)
             .setProjectId(PROJECT_ID)
             .setDatabaseId(DATABASE_ID)
+            .setCredentials(NoCredentials.getInstance())
+            .setHost("http://localhost:" + PORT);
+
+    options2 =
```

Comment:
Is `options2` only for testCustomChannelAndCredentials()? Do we need both `options` and `options2` to cover all the test cases? If not, could we rename them to be more meaningful?

Reply:
inlined to the specific test